### DISCUSSION TITLE
Disaster Recovery Cluster

### DISCUSSION THREAD

**Discussion by simon-appleyard** [ravendb/ravendb#14488] [General]
Hello,

In a 3 node cluster configuration, are there any problems with having 2 nodes in one data centre, and 1 node in another data centre? This would form a DR configuration. 

In this situation would it make sense for the DR node to be a Watcher node? The DR node would never want to be the Leader would it? If so, would we ideally need 3 nodes in 1 data centre and 1 node in the other?

Or would it be best practice for all nodes to be in the same data centre?

Many Thanks

**Comment by garayx** [comment]
Hi 

A typical deployment of RavenDB is a cluster which consists of 3 nodes. In addition to that you can also have a separate RavenDB instance (not part of that cluster) and set up an external replication to it.

See the following chapters in Oren's book:

- [https://ravendb.net/learn/inside-ravendb-book/reader/4.0/15-production-deployments](https://ravendb.net/learn/inside-ravendb-book/reader/4.0/15-production-deployments)
- [https://ravendb.net/learn/inside-ravendb-book/reader/4.0/16-monitoring-troubleshooting-and-disaster-recovery](https://ravendb.net/learn/inside-ravendb-book/reader/4.0/16-monitoring-troubleshooting-and-disaster-recovery)

**Reply by georgiosd** [reply]
@simon-appleyard also extensive discussion here: https://github.com/ravendb/ravendb/discussions/14150
